sdk/android/skin/src/main/java/com/ooyala/android/skin/OoyalaSkinLayoutController.java

```diff
@@ -528,6 +532,48 @@ public void switchVRMode(VrMode vrMode) {
     }
   }
 
+  @Override
+  public void onLayoutChange(View v, int left, int top, int right, int bottom, int oldLeft, int oldTop, int oldRight, int oldBottom) {
+    if (rootRecyclerView == null) {
+      return;
+    }
+    for (String tag: getTagsForScrollableViews()) {
```

Comment:
Let's add `check on null` for `getTagsForScrollableViews()` and smth like this in that case: `DebugMode.LogW(TAG, "Warning, tags array is not initialized")`

Reply:
Check added, don't agree we need log message here. There is no point to implement setRootRecyclerView with null returned in getTagsForScrollableViews and if it return null after all , its better does not slow callback onLayoutChange() with logging call.

Reply:
Ok. agree.